content/en/integrations/guide/aws-integration-with-terraform.md

```diff
@@ -0,0 +1,175 @@
+---
+title: The AWS Integration with Terraform
+kind: documentation
+further_reading:
+- link: "https://www.datadoghq.com/blog/managing-datadog-with-terraform/"
+  tag: "Blog"
+  text: "Managing Datadog with Terraform"
+- link: "https://docs.datadoghq.com/integrations/faq/how-to-import-datadog-resources-into-terraform/"
+  tag: "FAQ"
+  text: "Importing Datadog resources into Terraform"
+---
```

Comment:
Because we're moving this page, we'll need to add an alias to the front matter as well.